src/components/sideNavigation/SideNavigationLoadingItem.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import * as className from 'classnames';
+
+export interface ISideNavLoadingItemProps {
+  classes?: string;
```

Comment:
you should use the same prop name everywhere, `className` or classes, `className` is the one used by React so i would pick this one.

Reply:
this component should be written as a pure function too